fastddsspy_participants/test/model/blackbox/EndpointDatabaseTest.cpp

```diff
@@ -0,0 +1,351 @@
+// Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
In general, apply same comments as in ParticipantDatabaseTests:

- Use a smaller sleep time, and use them all from a const variable set in the test header (upper part of the test file inside `test` namespace)
- Do endpoint fill information from a common function that allow to create "random" (arbitrary) ones
- check the n endpoints test, as it does not do what it must